TGC/decisions/ENG.txt

```diff
@@ -4809,20 +4809,43 @@ political_decisions = {
 					who = NET
 					value = 60
 				}
+				NET = {
+					NOT = { has_country_modifier = willem_i_absolutism }
```

Comment:
Nitpick: If a flag is going to be exposed to the player (such as in allow tags) make sure it has a localization (I've started the effort in 00_flags.csv)